presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/MetadataDao.java

```diff
@@ -213,6 +214,15 @@ int dropView(
 
     @SqlQuery("SELECT temporal_column_id\n" +
             "FROM tables\n" +
-            "WHERE table_id = :tableId")
+            "WHERE table_id = :tableId\n" +
+            "AND temporal_column_id IS NOT NULL")
     Long getTemporalColumnId(@Bind("tableId") long tableId);
+
+    @VisibleForTesting
```

Comment:
I'm not sure this annotation is necessary. It seems like something the DAO should support anyway (and we might use in the future).